website/blog/2021-06-04-running-presto-alluxio-sds.md

```diff
@@ -0,0 +1,365 @@
+
+        -   `etc/config.properties`:
+
+            ```
+            coordinator=true
+            node-scheduler.include-coordinator=true
+            http-server.http.port=8080
+            query.max-memory=2GB
+            query.max-memory-per-node=1GB
+            discovery-server.enabled=true
+            discovery.uri=http://localhost:8080
+            ```
+
+        -   `etc/jvm.config`:
+
+            ```
+            -server
+            -Xmx4G
+            -XX:+UseConcMarkSweepGC
+            -XX:+ExplicitGCInvokesConcurrent
+            -XX:+CMSClassUnloadingEnabled
+            -XX:+AggressiveOpts
+            -XX:+HeapDumpOnOutOfMemoryError
+            -XX:OnOutOfMemoryError=kill -9 %p
+            -XX:ReservedCodeCacheSize=150M
+            ```
+
+        -   `etc/log.properties`
+
+            ```
+            com.facebook.presto=INFO
+            ```
+
+        -   `etc/catalog/hive.properties`
+
+            ```
+            connector.name=hive-hadoop2
+            hive.metastore.uri=thrift://localhost:9083
+            ```
+
+    3.  Start Presto server:
+
+        `bin/launcher start`
+
+    4.  Launch the Presto CLI client:
+
+        1.  Download it from Maven Central:
+
+            `wget https://repo1.maven.org/maven2/com/facebook/presto/presto-cli/0.253/presto-cli-0.253-executable.jar`
+
+        2.  Set executable permission:
+
+            `chmod +x presto-cli-0.253-executable.jar`
+
+        3.  Start the CLI:
+
+            `./presto-cli-0.253-executable.jar --catalog hive --schema test`
+
+        4.  Inside the CLI, verify that the test database is accessible from Presto:
+
+            ```SQL
+            show schemas from hive;
+            show tables from hive.test;
+            select * from hive.test.person;
+            select count(*) from person;
+            ```
+
+-   ### Alluxio
+
+    Download the Alluxio distribution tarball and extract the files. Inside the Alluxio directory:
+
+    1.  Create Alluxio config file from template:
+
+        `cp conf/alluxio-site.properties.template conf/alluxio-site.properties`
+
+    2.  Edit `conf/alluxio-site.properties`:
+   
+        Uncomment the lines with `alluxio.master.hostname` and `alluxio.master.mount.table.root.ufs`, 
+        so that `alluxio.master.hostname` is set to `localhost` 
+        and `alluxio.master.mount.table.root.ufs` is set to `${alluxio.work.dir}/underFSStorage`
+
+    3.  Mount RAMFS filesystem:
+
+        `./bin/alluxio-mount.sh SudoMount`
+
+        You may need to input your password because mounting RAMFS requires sudo privileges.
+
+    4.  Format Alluxio filesystem:
+
+        `./bin/alluxio format`
+
+        NOTE: This step is only required when you run Alluxio for the first time. If you run this command for an existing Alluxio cluster, all previously stored data and metadata in the Alluxio filesystem will be erased. However, data in under storage will not be changed.
```

Comment:
"data in under storage will not be changed."  I guess you mean the data in the underlying storage would not be changed,  just wanna clarify, is the underlying storage here hive data files or hive-metastore, or both?

Reply:
I think both won't be affected.